Assets/2_MainScene/S2_Scripts/S2_PlayerScripts/S2_PlayerCollisionController.cs

```diff
@@ -17,50 +17,82 @@ public class S2_PlayerCollisionController : MonoBehaviour {
     void Start() { Instance = this; }
 
     void FixedUpdate() {
-        foreach (Vector3 item in dir) {
-            if (Physics.Raycast(transform.position, item, out RaycastHit hit, 0.5f, 1 << 8)) {
+        for(int i = 0; i < dir.Length; i++) { 
+            if (Physics.Raycast(transform.position, dir[i], out RaycastHit hit, 0.5f, 1 << 8)) {
                 transform.Translate(hit.normal / 4.5f);
-                PlaySparks(hit);
+                PlaySparks(hit.point);
             }
-            if (Physics.Raycast(transform.position, item, 10.0f, 1 << 9) && playerStatus[0]) S2_PlayerAudioController.Instance.PlayRandomWhoosh();
+            if (Physics.Raycast(transform.position, dir[i], 10.0f, 1 << 9) && playerStatus[0]) S2_PlayerAudioController.Instance.PlayRandomWhoosh();
         }
 
-        if (!playerStatus[0] || !playerStatus[1] || playerStatus[2]) return;
-        foreach (LayerMask item in layers) {
-            if (Physics.BoxCast(collider.bounds.center, collider.bounds.extents / 2, Vector3.forward, out RaycastHit hit, transform.rotation, 2.1f, 1 << item)) {
-                ScoreLimiter();
-
-                if (item == 9) {
-                    if (S2_HUDUI.Instance.GetAttributes()[0].value > 0) S2_HUDUI.Instance.SetAttributes(0, -1);
-                    else if (S2_HUDUI.Instance.GetAttributes()[1].value > 1) S2_HUDUI.Instance.SetAttributes(1, -1);
-                    else {
-                        S2_HUDUI.Instance.SetAttributes(1, -1);
-                        S2_HUDUI.Instance.SendGameInfo();
-                        S2_PlayerController.Instance.PlayerDeath();
-                        S2_PlayerAudioController.Instance.PlayAudio(0);
-                        Instantiate(playerDeathParticles, transform.position, Quaternion.identity);
-
-                        playerStatus[1] = false;
-                    }
-
-                    PlaySparks(hit);
-                    ToggleWasHit();
-                    Invoke(nameof(ToggleWasHit), 0.25f);
+        //if (!playerStatus[0] || !playerStatus[1] || playerStatus[2]) return;
```

Comment:
We're never going to be going back to this code, might as well delete it fully instead of commenting it out